index.html

```diff
@@ -74,148 +74,9 @@ <h1 class="hero__title">
-              <li>css</li>
-              <li>javaScript</li>
-            </ul>
-            <div class="btn-card">
-              <button class="btn btn-primary" type="button">See Project</button>
-            </div>
-          </div>
-        </div>
-        <div class="card-project">
-          <div class="card__image">
-            <img
-              class="card__img"
-              src="./images/Project3.png"
-              alt="avocode page"
-            />
-          </div>
-          <div class="card__body">
-            <h2 class="card__title">Tonic</h2>
-            <ul class="card__information">
-              <li class="card__information-name">CANOPY</li>
-              <li>
-                <img src="./images/counter.svg" alt="information separator" />
-              </li>
-              <li class="card__information-position">Back End Dev</li>
-              <li>
-                <img src="./images/counter.svg" alt="information separator" />
-              </li>
-              <li class="card__information-year">2015</li>
-            </ul>
-            <p class="card__description">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <ul class="card__languages">
-              <li>html</li>
-              <li>css</li>
-              <li>javaScript</li>
-            </ul>
-            <div class="btn-card">
-              <button class="btn btn-primary" type="button">See Project</button>
-            </div>
-          </div>
-        </div>
-        <div class="card-project">
-          <div class="card__image card__swap-r">
-            <img
-              class="card__img"
-              src="./images/Project4.png"
-              alt="avocode page"
-            />
-          </div>
-          <div class="card__body card__swap-l">
-            <h2 class="card__title">Multi-Post Stories</h2>
-            <ul class="card__information">
-              <li class="card__information-name">CANOPY</li>
-              <li>
-                <img src="./images/counter.svg" alt="information separator" />
-              </li>
-              <li class="card__information-position">Back End Dev</li>
-              <li>
-                <img src="./images/counter.svg" alt="information separator" />
-              </li>
-              <li class="card__information-year">2015</li>
-            </ul>
-            <p class="card__description">
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <ul class="card__languages">
-              <li>html</li>
-              <li>css</li>
-              <li>javaScript</li>
-            </ul>
-            <div class="btn-card">
-              <button class="btn btn-primary" type="button">See Project</button>
-            </div>
-          </div>
-        </div>
+      <section id="works" class="works"> 
       </section>
+    
```

Comment:
- [ ] @Team great work here, however, there is a slight error in your desktop design, kindy make sure the edges of your paragraph and buttons align with the image edges like in the Figma design:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/168375337-be0f331c-9903-46ae-929c-9513603eca1a.png) |  <img width="485" alt="image" src="https://user-images.githubusercontent.com/61408860/168375565-81e9922d-7ad5-459c-9dc6-57e8ddfd0e1f.png">

Reply:
I've Just done it. Thank you for your feedback